packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -100,10 +96,6 @@ describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvid
 					mockConfigProvider(settings),
 				);
 
-			let summary2Error: any;
```

Comment:
The summarizing container should still close right?

Reply:
No because it was the summarizer who won. Its summary got acked

We check below that summarizingContainer2 closes, summarizingContainer wins (I should have renamed it summarizingContainer1)

Reply:
I actually am not sure why it didn't get closed and summarizingContainer2 wasn't elected as the new official summarizing client

Reply:
Let me put in another way - Why remove this event handler? `summarizingContainer2` still closes so why not validate the error?

Reply:
Is it because now summarizer close raises a telemetry event not an error?

Reply:
The error is undefined, so there's not much to validate. Checking to see if summarzingContainer2 is closed is essentially the same thing as waiting for the closed event